spec/support/network_helpers.rb

```diff
@@ -43,7 +43,7 @@ def call_web_mock_function_with_agent_host_exclusions
   def check_availability_by_http_request(host, port)
     uri = URI("http://#{host}:#{port}/info")
     request = Net::HTTP::Get.new(uri)
-    request['X-Datadog-Untraced-Request'] = true
+    request[Datadog::Transport::Ext::HTTP::HEADER_DD_INTERNAL_UNTRACED_REQUEST] = true
```

Comment:
Should the value be `true` or string `'1'` ?

Reply:
String, thank you!